bin/pygrb/pycbc_pygrb_grb_info_table

```diff
@@ -88,8 +88,8 @@ headers.append('IFOs')
 for ifo in opts.ifos:
     antenna = Detector(ifo)
     factor = get_antenna_dist_factor(antenna,
-                                     numpy.deg2rad(float(opts.ra)),
-                                     numpy.deg2rad(float(opts.dec)),
+                                     float(opts.ra),
+                                     float(opts.dec),
```

Comment:
Unless I am missing some specific use case, the `--ra` and `--dec` args should be marked as `type=float`, and then you can drop the `float()` conversions here and above.